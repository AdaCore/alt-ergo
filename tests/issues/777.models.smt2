(set-logic ALL)
(set-option :produce-models true)
(declare-const i Int)
(define-fun C ((j Int)) Bool (> j 0))
(assert (C i))
(check-sat)
(get-model)
