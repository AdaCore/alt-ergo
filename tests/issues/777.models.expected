
unknown
(
  (define-fun i () Int 2)
)
